pkg/chain/ethereum/wei_test.go

```diff
@@ -84,7 +84,7 @@ func TestUnmarshalTextValue(t *testing.T) {
 			expectedError: fmt.Errorf("failed to parse value: [3 wei ]"),
 		},
 
```

Comment:
The blank line is probably not needed.

Reply:
98cf58736765ea8446021bd24371789574580879